### PR TITLE
Unflag View Transition support

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- Remove `experimental.viewTransitions` flag.
- RFC was approved and merged

## Testing

- Tests updated to not have the flag.

## Docs

- https://github.com/withastro/docs/pull/4320

### CODE REVIEW

**Review by ematipico — APPROVED** [review]
Don't we have the CLI flag to remove too?

**Comment by matthewp** [comment]
@ematipico No, I forgot to add one. It looks like we have forgotten to add the CLI flags for experimental features for the last few.

**Review by bluwy — APPROVED** [review]

File: .changeset/grumpy-pens-melt.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { defineConfig } from 'astro'
+
+export default defineConfig({
+-  experimental: {
+-    viewTransitions: true,
+-  }
+})
+```
+
+See the [View Transitions guide](https://docs.astro.build/en/guides/view-transitions/) to learn about how to use the API.
```
Comment (sarah11918) [inline .changeset/grumpy-pens-melt.md:19]:
```suggestion
After removing this flag, please also consult the specific [upgrade to v3.0 advice](https://docs.astro.build/en/guides/view-transitions/#upgrade-to-v30-from-v2x) as some API features have changed and you may have breaking changes with your existing view transitions.

See the [View Transitions guide](https://docs.astro.build/en/guides/view-transitions/) to learn how to use the API.
```

**Review by sarah11918 — APPROVED** [review]
Just shouting out that because of the API changes, they should also see the upgrading section in particular!

After that.... let's gooooo!

**Pull request merged**